sample/src/main/kotlin/com/slack/circuit/sample/home/HomePresenter.kt

```diff
@@ -162,16 +155,16 @@ fun HomeContent(state: HomeScreen.State, eventSink: (HomeScreen.Event) -> Unit)
       .collect { isVisible ->
         // Toggle if state says the modal should be visible but the snapshot says it isn't.
         if (state.petListFilterState.showBottomSheet && !isVisible)
-          eventSink(PetListFilterEvent(PetListFilterScreen.Event.ToggleAnimalFilter))
+          state.eventSink(PetListFilterEvent(PetListFilterScreen.Event.ToggleAnimalFilter))
       }
   }
 
   ModalBottomSheetLayout(
     sheetState = modalState,
     sheetContent = {
       Column {
-        GenderFilterOption(state.petListFilterState, eventSink)
-        SizeFilterOption(state.petListFilterState, eventSink)
+        GenderFilterOption(state.petListFilterState, state.eventSink)
+        SizeFilterOption(state.petListFilterState, state.eventSink)
```

Comment:
Hmm this new approach makes possible something I hadn't considered to this point: we _could_ pass `state.petListFilterState.eventSink` here instead of `state.eventSink`. This would completely bypass the homeNavPresenter entirely (when processing an event) and simplify things slightly.

I'm not entirely sure how I feel about it. This would eliminate the need to wrap sub-screen events as we currently do which is a bonus